amr-wind/core/Field.H

```diff
@@ -318,6 +318,10 @@ public:
 
     void fillpatch(amrex::Real time) noexcept;
     void fillpatch(amrex::Real time, amrex::IntVect ng) noexcept;
+    void fillpatch(
```

Comment:
Seems like the least friction way to achieve what you want. My only suggestion would be to name this method differently (e.g., `fillpatch_sibling_fields` or something along those lines) to indicate in an explicit way that this method is doing something different than what the other `fillpatch` methods are doing.

Reply:
Good idea. Done.